addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/handler/LoxoneMiniserverHandler.java

```diff
@@ -411,11 +423,18 @@ private void addChannel(List<Channel> channels, String itemType, ChannelTypeUID
         } else if (control instanceof LxControlInfoOnlyDigital) {
             addChannel(channels, "Switch", roSwitchTypeId, id, label, "Digital virtual state", tags);
         } else if (control instanceof LxControlInfoOnlyAnalog) {
-            addChannel(channels, "Number", roAnalogTypeId, id, label, "Analog virtual state", tags);
+            if (addChannel(channels, "Number", roAnalogTypeId, id, label, "Analog virtual state", tags)) {
+                setStateDescription(id, ((LxControlInfoOnlyAnalog) control).getFormatString(), true, null, 0);
```

Comment:
Could you explain a bit more what you mean?

Reply:
To me it would make sense if the different types of control objects are in charge of expressing how the state description should look like instead of having that in this handler class, is that more clear so the parent class of the control object could have a method constuctStateDescription